.github/workflows/sync_project.yml

```diff
@@ -0,0 +1,30 @@
+name: Sync qube project
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+        - name: Setup Python
+          uses: actions/setup-python@v1
+          with:
+            python-version: 3.8
+
+        - name: Install qube
+          run: pip install qube
+
+        - uses: actions/checkout@v2
+          with:
+              fetch-depth: 0
+          name: Check out source-code repository
+
+        - uses: oleksiyrudenko/gha-git-credentials@v1
+          with:
+               name: 'sven1103'
+               email: 'sven.fillinger@qbic.uni-tuebingen.de'
+               actor: 'sven1103'
+               token: '${{ secrets.QUBE_SYNC_TOKEN}}'
```

Comment:
Is this the standard then? Or does this need to be updated by everyone pulling the code?

Reply:
@jenniferboedker We will hardcode Johnny and you do no need to adapt anything.

Reply:
Ah ok, so Johnny is the developer of QBiC in Github?

Reply:
Johnny is the best and most hardworking developer of QBiC. 
He never complains and just does what he is supposed to do (most of the times).